openfecwebapp/templates/macros/filters/range.html

```diff
@@ -1,21 +1,19 @@
 {% macro amount(name, label) %}
 <fieldset class="filter" id="{{name}}-field">
   <legend class="label">{{ label }}</legend>
-  <div class="js-filter">
-    <div class="range">
-      <div class="range__input range__input--min">
-        <label for="min_{{name}}">From</label>
-        <input id="min_{{name}}" type="text" name="min_{{name}}" data-range="min" data-prefix="$" data-suffix="or more" tabindex="0">
-      </div>
-      <div class="range__hyphen">-</div>
-      <div class="range__input range__input--max">
-        <label for="max_{{name}}">To</label>
-        <input id="max_{{name}}" type="text" name="max_{{name}}" data-range="max" data-prefix="$" data-suffix="or less" tabindex="0">
-      </div>
-      <button class="button--go button--standard" type="button">
-        <span class="u-visually-hidden">Search</span>
-      </button>
+  <div class="range">
+    <div class="range__input range__input--min js-filter" data-filter="range">
```

Comment:
So I ran into issues because each instance of a `Filter` expects the same `name`. Because the `min` and `max` filters have different names, the URL wasn't being set properly. I might be missing something, but I _think_ this is right to keep them separate.